.pre-commit-config.yaml

```diff
@@ -132,7 +132,7 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v9.0.4
+    rev: v8.0.19
```

Comment:
@gonzalesedwin1123 what was the reason for reverting it?

Reply:
There are pre-commit errors generated by the new version @kneckinator

Reply:
Are you ok with the rollback @jeremi ?

Reply:
I'll merge this "as-is" anyway. We can discuss the version later.